src/providers/trusted_service/context/asym_sign.rs

```diff
@@ -25,14 +26,15 @@ impl Context {
         Ok(signature)
     }
 
-    pub fn asym_verify(
```

Comment:
Is the missing doc lint activated for this module?

Reply:
I think not because it's not included in the feature list that we use with `clippy` (or even `cargo build`).

Reply:
Plus, I'm unsure whether it will complain about missing docs for things that are not public for the "library". This isn't public beyond the provider module

Reply:
Ahhh gotcha